repository-data/application/src/main/resources/hcm-config/configuration/update/Migration20180319UpcomingPublicationsSearchable.groovy

```diff
@@ -0,0 +1,33 @@
+package org.hippoecm.frontend.plugins.cms.admin.updater
+
+import org.hippoecm.repository.util.JcrUtils
+import org.onehippo.repository.update.BaseNodeUpdateVisitor
+
+import javax.jcr.Node
+
+/**
+ * This migration sets all the documents passed in to searchable
+ */
+class Migration20180319UpcomingPublicationsSearchable extends BaseNodeUpdateVisitor {
+
+    private static final String PROPERTY_SEARCHABLE = "common:searchable"
+
+    boolean doUpdate(Node node) {
+        try {
+            log.info("attempting to migrate node: " + node.getPath())
+
+            JcrUtils.ensureIsCheckedOut(node)
+            node.setProperty(PROPERTY_SEARCHABLE, true)
```

Comment:
is it worth having a more generic script - "Set Searchable Flag" - with a parameter to set it to `true` or `false`?

Reply:
As discussed don't think it's worth it in this case